lib/pages/bootstrap/tom_bootstrap_dialog.dart

```diff
@@ -32,8 +38,13 @@ class TomBootstrapDialog extends StatefulWidget {
 
 class TomBootstrapDialogState extends State<TomBootstrapDialog> {
   final _saveRecoveryWordsInteractor = getIt.get<SaveRecoveryWordsInteractor>();
+  final _deleteRecoveryWordsInteractor =
+      getIt.get<DeleteRecoveryWordsInteractor>();
   Bootstrap? bootstrap;
+
   String? titleText;
+  Widget? body;
+  final buttons = <AdaptiveFlatButton>[];
 
   UploadRecoveryKeyState _uploadRecoveryKeyState =
```

Comment:
Why don't you change `ValueNotifier<UploadRecoveryKeyState>`

Reply:
use `setState` instead